tests/Unit/Domain/Test_BlockAndElementLogicalCoordinates.cpp

```diff
@@ -703,7 +703,8 @@ void test_element_ids_are_uniquely_determined() {
 }
 
 void test_block_logical_coordinates_with_roundoff_error() {
-  const auto shell = domain::creators::Shell(1., 3., 0, {{3, 3}});
+  const auto shell = domain::creators::Sphere(
```

Comment:
`shell` -> `sphere` and on line 708

Reply:
the interior is excised here so I think keeping the variable name `shell` is fine